websites/A/AniWorld/presence.ts

```diff
@@ -49,73 +53,75 @@ presence.on(
 );
 
 presence.on("UpdateData", async () => {
+	if (!strings) strings = await getStrings();
```

Comment:
You shouldn't use getStrings if you don't plan on supporting multiLanguage. Either remove this and use static strings or add proper support for multiLanguage for all strings (I recommend the first option)

Reply:
Strings removed.